test/conftest.py

```diff
@@ -156,6 +161,17 @@ def observationer(firedb, sagsevent, observationstype, punkt):
 
 @pytest.fixture()
 def beregning(firedb, sagsevent, observationer):
+    sagsevent.event = EventType.BEREGNING
```

Comment:
Might not want to use this eventtype - See discussion at https://github.com/Kortforsyningen/FIRE-DDL/issues/37

Reply:
Thanks. I wasn't aware of that.